vars/envToPragmas.groovy

```diff
@@ -0,0 +1,18 @@
+/* groovylint-disable ParameterName */
+// vars/envToPragmas.groovy
+
+/**
+ * envToPragmas.groovy
+ *
+ * envToPragmas variable
+ */
+
+Map call() {
+    Map pragmas = [:]
+    if (env.pragmas)
+        pragmas = "${env.pragmas}"[1..-2].split(', ').collectEntries { entry ->
+            String[] pair = entry.split('=', 2)
```

Comment:
The previous version did
`entry.split('= ')`
So this version explicitly trims the value